src/clauses/sub-clauses/OrderBy.ts

```diff
@@ -72,16 +77,19 @@ export class OrderBy extends CypherASTNode {
 }
 
 class Skip extends CypherASTNode {
```

Comment:
Another solution could be to have `Offset` that extends Skip and removing the flag

Reply:
Unbiased honestly, I think we will have some cases similar in the future, where a GQL syntax will be an alias of an existing syntax, I would handle all the aliases in the same way, but we handle them just like this as well!